cypress/integration/blocks.js

```diff
@@ -596,5 +596,35 @@ if (Cypress.env('API') !== 'guillotina') {
     //   //   cy.contains(expected);
     //   // }
     // });
+
+    it.only('Add Table of Contents block', () => {
+      const h2 = 'expectedH2';
```

Comment:
@steffenri I prefer to be explicit here and not use a variable.

Reply:
changed.